tests/components/smartthings/test_climate.py

```diff
@@ -279,10 +278,8 @@ def air_conditioner_fixture(device_factory):
     await setup_platform(hass, CLIMATE_DOMAIN, devices=[buggy_thermostat])
     state = hass.states.get("climate.buggy_thermostat")
     assert state.state == STATE_UNKNOWN
-    assert (
-        state.attributes[ATTR_SUPPORTED_FEATURES]
-        == SUPPORT_TARGET_TEMPERATURE_RANGE | SUPPORT_TARGET_TEMPERATURE
-    )
+    assert (state.attributes[ATTR_SUPPORTED_FEATURES] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)